src/app/shared/FormProcessingStatus.ts

```diff
@@ -0,0 +1,34 @@
+export class FormProcessingStatus {
+  /**
+   * Indicates weather the form is being processed / loaded
+   */
+  processing = true;
+
+  /**
+   * Success message.
+   */
+  success: string;
+
+  /**
+   * Success message.
+   */
+  error: string;
+
+  constructor(processing: boolean) {
+    this.processing = processing;
+  }
+
+  public begin() {
+    this.processing = true;
+  }
+
+  public done(params: { successMessage?: string, errorMessage?: string, error?: any } = {}) {
+    this.success = params.successMessage;
+    this.error = params.errorMessage;
+    if (params.error) {
+      console.log(params.error);
```

Comment:
Is this a debug artefact? If not, we should mention in the log message why we logged it (Console outputs like `Object{condition: '?!?!', type: 'unknown'}` are not helpful if you don't know were to look 😉)

Reply:
This was actually intended - in case of an unknown error it's nice to check the console for further debug information...but i understand that it can be confusing in the log output...should I delete it then?

Reply:
No, let's keep it there for the moment. I just saw it in the test output; in the browser, there is a reference to where it was logged, so that's what I originally was going for 😊